vulture_os/services/frontend/models.py

```diff
@@ -1360,6 +1360,8 @@ def generate_filebeat_conf(self, module_type):
 
         if module_type == "input" and self.filebeat_module != "_custom":
             return ""
+        if module_type == "module" and self.filebeat_module == "_custom":
+            return ""
```

Comment:
```suggestion
        if module_type in ["input", "module"] and self.filebeat_module != "_custom":
            return ""
```

Reply:
woops, didn't see the difference on condition for the second term, my proposed change is invalid